packages/repository/src/connectors/connector.ts

```diff
@@ -12,13 +12,53 @@ import {
 } from '../common-types';
 import {Model} from '../model';
 
+/**
+ * Interfaces adopted by a {@link Connector}.
```

Comment:
Could you explain more about the `@link` Connector?

Reply:
This is a tsdoc syntax to reference a TypeScript item/class/interface/etc.:

https://tsdoc.org/pages/tags/link/